string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,23 @@
+/* exported isVowel */
+/*
+- look at the input letter
+- change letter to lower case
+- compare against each vowel
+- if it is, true
+- if it isn't, false
+*/
+
+function isVowel(char) {
+  if (char.toLowerCase() === 'a') {
+    return true;
+  } else if (char.toLowerCase() === 'e') {
+    return true;
+  } else if (char.toLowerCase() === 'i') {
+    return true;
+  } else if (char.toLowerCase() === 'o') {
+    return true;
+  } else if (char.toLowerCase() === 'u') {
+    return true;
```

Comment:
Don't forget about your `||` operator! All these conditional blocks are doing the same work, so this would be a great opportunity to make use of that operator so that you can do all this with one `if` statement